frontend/cypress/e2e/climateProfiles.cy.js

```diff
@@ -0,0 +1,26 @@
+describe('Climate profiles page', function () {
+    beforeEach(function () {
+        cy.visit('http://localhost:5173/profiles')
```

Comment:
Is `http://localhost:5173` prefix required or can it be relative like: `/profiles`?

Reply:
Now it can be relative. I added http://localhost:5173 to cypress.config.js

Reply:
That'll work, will have to look at why Vite doesn't proxy these e2e requests.